### ISSUE TITLE
Name and icon [Request for comments]

### ISSUE THREAD

**Issue by williambl** [deltamc-project/deltamc-python#45]
There's a bit of a problem with the name/icon of the program:
 ## Name:
 * It's a bit too long.
 * It's [hard to Google](https://encrypted.google.com/search?hl=en&q=cman), as there's already a [piece of software with that name](https://sourceware.org/cluster/cman/)
 * So I propose mpm - minecraft package manager.

 ## Icon:
 * The orange lambda seems a bit half-life. I think we should replace it with a delta, as delta is often used to signify change or difference - and one of the main things the program does is upgrade mods - changing them.

I'd like to get everyones input on these ideas - I'll add what everyone thinks to this table:

| Username        | Program name | Program icon                |
|-----------------|--------------|-----------------------------|
| @LewisTehMinerz  | mcdelta/deltamc|[Impossible triangle of minecraft blocks](https://user-images.githubusercontent.com/10055034/29441293-415742fc-8398-11e7-81b4-06459752a35c.png)|
| @randomtestfive | deltamc|[Impossible triangle of minecraft blocks](https://user-images.githubusercontent.com/10055034/29441293-415742fc-8398-11e7-81b4-06459752a35c.png)|
| @Valiec         | deltamc| Grass block with a delta sign |
| @williambl      | mpm/mcdelta/deltamc|[Impossible triangle of minecraft blocks](https://user-images.githubusercontent.com/10055034/29441293-415742fc-8398-11e7-81b4-06459752a35c.png)|

**Comment by LewisTehMinerz** [comment]
mpm, delta sign with grass block

**Comment by randomtestfive** [comment]
Just found another project called [mcpm](https://github.com/mcpm/mcpm).

**Comment by randomtestfive** [comment]
This is a bit rough and maybe to high res for a program icon, but I think it could work. ![penrose-rough](https://user-images.githubusercontent.com/10055034/29441293-415742fc-8398-11e7-81b4-06459752a35c.png)

**Comment by williambl** [comment]
@randomtestfive I like that icon!

Regarding the mcpm, while it has a similar name, it hasn't been updated in a while - and a google for mpm doesn't show it.

**Comment by Valiec** [comment]
Regarding the logo, how about a delta on a grass block (i.e. the current logo with a delta instead of a lambda), rather than a grass block inside a delta.

**Comment by LewisTehMinerz** [comment]
@randomtestfive that icon looks awesome, I want to change my vote please

**Comment by randomtestfive** [comment]
Any suggestions for other blocks for the cleaner version?

**Comment by williambl** [comment]
@randomtestfive As it is, it already shows all the basic parts of minecraft - redstone ore shows mining and redstone, grass shows the world generation, and bookshelf shows building.

**Comment by randomtestfive** [comment]
Here's one with better outlines and a 64x64 version.
![penrose-cleaner](https://user-images.githubusercontent.com/10055034/29489518-e12c1a42-84f0-11e7-9c94-c2fa2f2f1d0f.png)
![penrose-cleaner-low](https://user-images.githubusercontent.com/10055034/29489521-e44d4b88-84f0-11e7-928d-6bfb9c88617d.png)

**Comment by williambl** [comment]
@randomtestfive Thanks! As there's a majority vote for your icon, I'm going to start using it. Now we just need to make an agreement on the name.

**Comment by LewisTehMinerz** [comment]
now that i think about it

mcdelta or deltamc

generally i am just throwing out random ideas

**Comment by williambl** [comment]
@LewisTehMinerz I like those!

**Comment by randomtestfive** [comment]
I like DeltaMC.

**Comment by williambl** [comment]
@Valiec what do you think about the name?

**Comment by Valiec** [comment]
I'm not 100% sure, but DeltaMC sounds good to me.

**Comment by williambl** [comment]
Ok, we are unanimous on deltamc. Googling it does produce [some small servers and youtube channels](https://encrypted.google.com/search?hl=en&q=deltamc), but they're all tiny and we should rise to the top of the list quickly.

**Comment by williambl** [comment]
I'm going to start changing it now.

**Comment by williambl** [comment]
Actually, github.com/deltamc and github.com/delta-mc exist. We can still call ourselves deltaMC, but we need a new username. I'm thinking 'delta-mc-pkg'? (delta mc package manager)

**Comment by williambl** [comment]
Actually, I'm going to change our username to deltamc-project.

**Comment by Valiec** [comment]
Ok. I'm going to go through the code and change all references to "CMAN" to "DeltaMC".

**Comment by williambl** [comment]
@Valiec already done in the new-syntax branch of deltamc-python.

**Comment by Valiec** [comment]
Just noticed, thanks! I was looking at the master branch.

**Comment by Valiec** [comment]
Sorry to reopen this, but I might be leaning towards MCDelta now. It sounds better to my ears (actually, I changed my mind a long time ago, but I didn't want to flip-flop on the name too fast, then I forgot about the project for nearly 2 years).

**Comment by williambl** [comment]
Yeah, I like that actually. Makes more sense to have MC at the start.

**Comment by williambl** [comment]
Seeing as MCDelta has been used for the discord and also the description of the support channel and everyone's OK with it, I've decided to change the other things to it as well. The actual source code name change can probably wait until curseforge support is done, just to avoid having two large merges at once.

**Comment by williambl** [comment]
I'm not touching the archive or the java repo though, the archive should be done after the code changes (so the URLs can be changed), and the java team can rename the java repo if they want to.
